python/tskit/drawing.py

```diff
@@ -551,28 +551,31 @@ def draw_y_axis(
             )
         if self.y_axis:
             y_axis.add(dwg.line((x, rnd(lower)), (x, rnd(upper))))
-            if tick_positions is not None:
-                for pos in tick_positions:
-                    tick = y_axis.add(
-                        dwg.g(
-                            class_="tick",
-                            transform=f"translate({x} {rnd(self.y_transform(pos))})",
-                        )
+            try:
+                tick_positions = ticks.keys()
+                tick_labels = ticks.values()
+            except AttributeError:
+                tick_positions = ticks
+                tick_labels = (f"{pos:.2f}" for pos in tick_positions)
+
+            for label, pos in zip(tick_labels, tick_positions):
```

Comment:
I don't see a good reason for unzipping the ticks dictionary into two lists here, we just have to zip them back together again.

Reply:
True - probably a hangover from when they were separate params. I'll change